.github/workflows/build-macos.yml

```diff
@@ -15,10 +15,10 @@ jobs:
       matrix:
         include:
           - arch: x64
-            runner: macos-latest
-          # https://github.blog/2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/
+            runner: macos-13
+          # https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/
```

Comment:
I think we can cut this line now that this matches the format of the other runners.

Reply:
Done.